Automattic/woocommerce-payments
# Added support to Cartes Bancaires

Fixes #8062 

#### Changes proposed in this Pull Request

> **Cartes Bancaires** is France’s local card network. More than 95% of these cards are co-branded with either Visa or Mastercard, meaning you can process these cards over either Cartes Bancaires or the Visa or Mastercard networks.

When these cards are used in Stores that do not support this branch, they default to Visa or Mastercard, otherwise, Stripe renders a dropdown with the brands.
To support CB, we nee you need to use EUR and use a French account. Other countries in the EEA require processing 50 EUR LIVE before Cartes Bancaires is enabled. Any country not part of the EEA does not support Cartes Bancaires, so the brands dropdown is never rendered.

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/a7959f16-abd5-4d46-a337-756129ac7fb3)

#### Testing instructions





- Make sure you are using a French Stripe account and processing EUR.
- Go to checkout and use one of the cobranded [test cards](https://docs.stripe.com/testing?testing-method=card-numbers#cards)
- Test saving and reusing saved cards
- Check the transactions page and the transactions in Stripe dashboard to ensure the correct brand is being used

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/febe607e-9499-4dab-a05d-27c026e27e5c)

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/8f14ebde-090b-4e48-871a-615ef74a6ebe)

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/8748a4ae-bae6-4aa2-8958-76693f837048)

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/c2834ad9-ca32-4da7-afc0-f4cf70128dd9)


**Known error:**
The **transactions list** does not show CB, it will only render Visa or Mastercard. I'm working on a fix for that from the server and it will **not** require another PR for the client.

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/2baa94da-91e4-4371-b49e-c6183e98c4a9)


-------------------


**Post merge**
